dart/math/Helpers.h

```diff
@@ -68,13 +65,22 @@ inline int delta(int _i, int _j) {
   return 0;
 }
 
-inline int sgn(double _a) {
-  if (_a < 0)
-    return -1;
-  else if (_a == 0)
-    return 0;
-  else
-    return 1;
+template <typename T> inline constexpr
+int sgn(T x, std::false_type)
```

Comment:
I considered to change it to `signum` with similar reason but just put it off. `sign` seems more clear and neat one than `signum`. I will change it to `sign` and deprecate `sgn`.